packaging/nntrainer.spec

```diff
@@ -347,6 +347,11 @@ NNSteamer tensor trainer static package for nntrainer to support inference.
 %define enable_cblas -Denable-blas=true
 %endif
 
+# enable encoder
+%if 0%{?use_encoder}
```

Comment:
This encoder depends on Applicaiton PicoGPT.. so.. please run the script when we build Applicatin PicoGPT

Reply:
Thank you for Review - Fix